discovery/pkg/openstack/reconciler.go

```diff
@@ -154,7 +154,22 @@ func (r *Reconciler) reconcile() {
 		r.reconcileSvcs(desiredSvcs, currentServices.Items)
 
 		desiredEndpoints := kubeEndpoints(r.BackendName, projectName, loadbalancers, pools)
+		for _, ep := range desiredEndpoints {
+			desiredEndpoints = append(desiredEndpoints, ep)
+		}
 		r.reconcileEndpoints(desiredEndpoints, currentEndpoints.Items)
+
+		// Log upstream services prometheus
+		r.Metrics.DiscovererUpstreamServicesMetric(r.BackendName, projectName, totalUpstreamServices)
+		r.Metrics.DiscovererInvalidServicesMetric(r.BackendName, projectName, totalUpstreamServices-len(loadbalancers))
```

Comment:
nit: took me some time to figure out how this subtraction results in the number of invalid services. Can we create a variable `invalidServicesCount` or something and move it up closer to `totalUpstreamServices` and `loadbalancers`?